Docs/sphinx_documentation/source/GPU.rst

```diff
@@ -333,6 +333,7 @@ Since CMake does not support autodetection of HIP compilers/target architectures
 yet, ``CMAKE_CXX_COMPILER`` must be set to a valid HIP compiler, i.e. ``hipcc`` or ``nvcc``,
 and ``AMReX_AMD_ARCH`` to the target architecture you are building for.
```

Comment:
At some point we could also re-consider that this variable takes right now a comma-separated list instead of `;`-separated (the latter is the std for CMake, the current & former is the pass-through we use for HIP's compiler option)